ios/Peepz/Peepz/Model/User.swift

```diff
@@ -0,0 +1,40 @@
+import Foundation
+
+struct User: Codable, Hashable {
```

Comment:
I couldn't use Codable in the end because Firebase is returning NSDictionary and not Json. Seems weird. Am I missing something ?

Reply:
Maybe you can ;)

https://medium.com/firebase-developers/swiftui-mapping-firestore-documents-using-swift-codable-589b6bfe462d

Reply:
The article provides example for Firebase.Firestore, we are currently using Firebase.Database. I know that we do want to migrate to firestore but until then I dont. think this can be done.